kotlin-analysis-clones/README.md

```diff
@@ -1,3 +1,64 @@
 # Kotlin-Analysis code clones
 
-Functionality related to code clones analysis in Kotlin projects
\ No newline at end of file
+Functionality related to code clones analysis in Kotlin projects
+
+## Run the project
+
+#### 1. Download list of repositories
+   
+You can use [seart-ghs.si.usi.ch]( https://seart-ghs.si.usi.ch/) (main language: Kotlin) for this purpose.
+The file has to be in csv format with ```name``` column, 
+containing repositories' names in the following format: ```username/project_name``` 
+
+#### 2. Clean duplicated repositories
+
+Run script for cleaning all duplicated repositories with different names: 
+``` shell script
+python3 scripts/clean_duplicates.py /path/to/csv_file/results.csv /path/to/cleaned/data/dir
+```
+   You can use ```--save_metadata``` flag to download metadata about all projects. Metadata includes information about repository's full name, owner, etc.
+   This script makes requests to GitHub API, so you should add your GitHub Token to environment variables (variable name is ```GITHUB_TOKEN```).
+
+#### 3. Load dataset 
+
+Run the following command to download the dataset:
+
+``` 
+python3 scripts/load_dataset.py /path/to/cleaned/data/dir/results.csv /path/to/dataset/dir --allowed_extensions kt
+```
+
+#### 4. Extract methods from dataset
+
+``` 
+python3 scripts/clones_batch_processing.py /path/to/dataset/dir /path/to/extracted_methods/dir 
+```
+
+You can use ```--batch_size``` option (default value is 300). 
+Also there is ```--start_from n``` option to start from batch with given number.
```

Comment:
Please, use the same notation. I mean: ```--batch_size n``` and ```--start_from n```, or ```--batch_size``` and ```--start_from```

Moreover, for the Python CLI arguments usually is used `-` instead of `_`. For example, ```--start-from```